src/Form/_FormCheckbox.scss

```diff
@@ -0,0 +1,157 @@
+.pgn__form-checkbox {
+  font-size: $font-size-base;
+  line-height: $line-height-base;
+  display: block;
+  flex-shrink: 0;
+  input {
+    border: 0px;
+    clip: rect(0px, 0px, 0px, 0px);
```

Comment:
So, in finding out what the `clip` property is, I also found out that it is deprecated, and we should maybe use `clip-path`? https://css-tricks.com/clipping-masking-css/#the-new-clip-path
But idk because IE11.

Reply:
clip-path doesn't look like it's well supported :/  Maybe we wait until we drop ie11 support and then style the input directly like Bootstrap 5 does.